apps/codelab/src/app/codelabs/index/index.component.html

```diff
@@ -50,18 +50,18 @@ <h2>
       of <a href="https://angular.io">Angular</a>!
     </h2>
     <div class="play">
-      <button class="learn-box typescript" routerLink="/typescript/0">
+      <button class="learn-box typescript" [routerLink]="['/typescript', 'intro']">
         <span aria-hidden="true">▶</span>
         <div>
           <div class="title" i18n>Learn TypeScript</div>
           <div class="hint" i18n>Skip if you're familiar with TypeScript</div>
         </div>
-        <img src="/assets/images/ts.png" alt="" class="" />
+        <img src="/assets/images/ts.png" alt="" class=""/>
       </button>
-      <button class="learn-box" routerLink="/create-first-app/0">
+      <button [routerLink]="['/angular', 'create-first-app', 'intro']" class="learn-box">
```

Comment:
If you replace 'intro' with '0' it'll start breaking because of the change in the `slide-routes.ts` file

Reply:
No, it doesn't. At least I didn't notice while testing. May in some special part of the application, that I didn't check
Anyway, I return the code in slide-routes.ts back